packages/office-ui-fabric-react/src/components/GroupedList/GroupHeader.scss

```diff
@@ -52,15 +57,14 @@ $groupHeader-ease-in-back: cubic-bezier(0.600, -0.280, 0.735, 0.045);
 
 .title {
   @include ms-font-xl;
-  padding: 14px 6px;
-  display: inline-block;
   cursor: pointer;
   outline: 0;
+  padding-left: 6px;
 }
 
 .expand {
   width: 36px;
-  height: 40px;
+  padding-top: 4px;
```

Comment:
Good point on the RTL for asymmetric horizontal padding… I think it can do fine without that. However I don’t recommend we get rid of the top padding. The world is not perfectly mathematical when it comes to visuals, and without it, the code may be slightly more elegant, but the product will be wonky. See this example:
![image](https://user-images.githubusercontent.com/20363037/27698265-6dc5fc7a-5cab-11e7-8dec-fdeac3863312.png)


As you can see, without top padding, the chevron looks misaligned towards the top, like it has a lump under its bed. There are a few things going on here.

- The text is optically centered based on the lower x-height, and not the capital or entire size of the text control.

- The chevron visually weights away from the point, and the glyph itself is set higher in its canvas. Both of these things make the chevron even appear higher than the circle check that it’s supposed to be aligned with. This is the only icon we will use here.

If I remember correctly, top padding has no bearing on RTL, so I really think we can have this one line of padding so our product doesn’t look weird. I’m not too worried about this causing any issues with future work, as it’s a small isolated case that we need to deal with these typographic considerations.

Here's a closer look, with the top being without top padding.
![image](https://user-images.githubusercontent.com/20363037/27698331-a9a660b8-5cab-11e7-9d96-0e08368c6fff.png)